src/ContrastRestClient/Http/TraceFilter.cs

```diff
@@ -47,6 +47,8 @@ public class TraceFilter
         public List<string> VulnTypes { get; set; }
         public List<string> AppVersionTags { get; set; }
         public List<long> ServerIds { get; set; }
+        public Boolean BeingTracket { get; set; }
```

Comment:
I think you want a "d" on the end here:
BeingTracket -> BeingTracked

Reply:
Yes, typo error, thanks for notice.